lms/djangoapps/courseware/views/views.py

```diff
@@ -2275,3 +2275,12 @@ def get_learner_username(learner_identifier):
     learner = User.objects.filter(Q(username=learner_identifier) | Q(email=learner_identifier)).first()
     if learner:
         return learner.username
+
+
+@api_view(['GET'])
+def is_mfe_search_enabled(request, course_id=None):
```

Comment:
let's call this courseware_mfe_search_enabled too so when future dev searches for that they get the whole stack 😆

Reply:
Updated. Now they'll get the "full" stack 🦆

Reply:
This is needed because we are having the `course_id` being parsed in the URL so the endpoint is expected to have this parameter.